docs/examples/test-app/application.yaml

```diff
@@ -0,0 +1,123 @@
+apiVersion: app.k8s.io/v1beta1
+kind: Application
+metadata:
+  name: "test-application-01"
+  labels:
+    app.kubernetes.io/name: "test-01"
+spec:
```

Comment:
Let's encourage to use descriptor, especially once it's integrated with kubectl/kubernetes-dashboard that would be the first experience user get and missing fields wouldn't look good on those tools.